packages/parser/src/index.ts

```diff
@@ -1,5 +1,5 @@
 import EventEmitter from 'eventemitter3';
-import moment from 'moment';
+import moment from 'moment-timezone';
```

Comment:
This import has actually always been incorrect and only working because of the way moment-timezone monkey patches the base moment object